etc/inc/whitelist-usr-share-common.inc

```diff
@@ -41,6 +41,8 @@ whitelist /usr/share/misc
 whitelist /usr/share/Modules
 whitelist /usr/share/myspell
 whitelist /usr/share/p11-kit
+whitelist /usr/share/perl
+whitelist /usr/share/perl5
```

Comment:
A lot profiles has `whitelist /usr/share/perl5`. I'm not sure whether we should move it to wusc or keep it in profiles.

Reply:
Do you want me to add it in the profile or keep it open for discussion?

Reply:
open for discussion

Reply:
If we go with this, `whitelist /usr/share/perl*` should be removed from profiles. @others opinions?

Reply:
I would say, add perl to wusc and remove the whitelist from profiles using it.

Reply:
disable-interpreters.inc `blacklist`s `/usr/share/perl*`, should be fine to allow it in wusc.